frontend/src/System/Tasks/Queued/QueuedTaskRow.js

```diff
@@ -200,7 +200,7 @@ class QueuedTaskRow extends Component {
           {
             clientUserAgent ?
               <span className={styles.userAgent} title={translate('TaskUserAgentTooltip')}>
-                {translate('from')}: {clientUserAgent}
+                {translate('From')}: {clientUserAgent}
```

Comment:
Should this really be capitalized?

Reply:
will the localization work with `"from"` in the json? - it is case sensitive is it not?

Reply:
Keys are camelcase, but the value is what I'm asking... should it be `from` instead of `From`. Right now in en.json it's defined with capital, assuming that is not used elsewhere should it be lower. If it is used elsewhere you'd have to add another key

Reply:
not used elsewhere; will push lower

  C:\Development\Code\Radarr\src\*cs
Search "translate('from')" (0 hits in 0 files of 2161 searched)

  C:\Development\Code\Radarr\frontend\src\*.js
Search "translate('from')" (1 hit in 1 file of 932 searched)
  C:\Development\Code\Radarr\frontend\src\System\Tasks\Queued\QueuedTaskRow.js (1 hit)
	Line 203:                 {translate('From')}: {clientUserAgent}